dev/test-studio/sanity.cli.ts

```diff
@@ -13,12 +27,34 @@ export default defineCliConfig({
   // A) `SANITY_STUDIO_REACT_STRICT_MODE=false pnpm dev`
   // B) creating a `.env` file locally that sets the same env variable as above
   reactStrictMode: true,
-  reactCompiler: {target: '18'},
+  reactCompiler: millionLintEnabled
+    ? {
+        target: '18',
+        sources: (filename) => {
+          if (filename.includes('node_modules')) {
```

Comment:
This is the default behavior of `sources`, since we're overriding it we have to ensure we don't accidentally try running the compiler on non-src files from npm.

Reply:
Would love to have this as a code comment too :innocent:

Reply:
Will do!